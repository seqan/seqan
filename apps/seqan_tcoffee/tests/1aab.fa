>hmgl_trybr
KKDSNAPKRAMTSFMFFSSDFRSKHSDLSIVEMSKAAGAAWKELGPEERKVYEEMAEKDKERYKREM
>hmgt_mouse
KPKRPRSAYNIYVSESFQEAKDDSAQGKLKLVNEAWKNLSPEEKQAYIQLAKDDRIRYDNEMKSWEEQMA
E
>hmgb_chite
ADKPKRPLSAYMLWLNSARESIKRENPDFKVTEVAKKGGELWRGLKDKSEWEAKAATAKQNYIRALQEYE
RNGG
>hmgl_wheat
DPNKPKRAPSAFFVFMGEFREEFKQKNPKNKSVAAVGKAAGERWKSLSESEKAPYVAKANKLKGEYNKAI
AAYNKGESA