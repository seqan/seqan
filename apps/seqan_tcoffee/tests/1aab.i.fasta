Scoring parameters:
*Gap opening: -13
*Gap extension: -1
*Scoring matrix: 
   A,B,C,D,E,F,G,H,I,J,K,L,M,N,O,P,Q,R,S,T,U,V,W,Y,Z,X,*,
A: 4,-2,0,-2,-1,-2,0,-2,-1,-1,-1,-1,-1,-2,0,-1,-1,-1,1,0,0,0,-3,-2,-1,0,-4
B: -2,4,-3,4,1,-3,-1,0,-3,-4,0,-4,-3,3,-1,-2,0,-1,0,-1,-1,-3,-4,-3,1,-1,-4
C: 0,-3,9,-3,-4,-2,-3,-3,-1,-1,-3,-1,-1,-3,-2,-3,-3,-3,-1,-1,-2,-1,-2,-2,-3,-2,-4
D: -2,4,-3,6,2,-3,-1,-1,-3,-4,-1,-4,-3,1,-1,-1,0,-2,0,-1,-1,-3,-4,-3,1,-1,-4
E: -1,1,-4,2,5,-3,-2,0,-3,-3,1,-3,-2,0,-1,-1,2,0,0,-1,-1,-2,-3,-2,4,-1,-4
F: -2,-3,-2,-3,-3,6,-3,-1,0,0,-3,0,0,-3,-1,-4,-3,-3,-2,-2,-1,-1,1,3,-3,-1,-4
G: 0,-1,-3,-1,-2,-3,6,-2,-4,-4,-2,-4,-3,0,-1,-2,-2,-2,0,-2,-1,-3,-2,-3,-2,-1,-4
H: -2,0,-3,-1,0,-1,-2,8,-3,-3,-1,-3,-2,1,-1,-2,0,0,-1,-2,-1,-3,-2,2,0,-1,-4
I: -1,-3,-1,-3,-3,0,-4,-3,4,3,-3,2,1,-3,-1,-3,-3,-3,-2,-1,-1,3,-3,-1,-3,-1,-4
J: -1,-4,-1,-4,-3,0,-4,-3,3,3,-3,3,2,-3,-1,-3,-3,-3,-2,-1,-1,2,-3,-1,-3,-1,-4
K: -1,0,-3,-1,1,-3,-2,-1,-3,-3,5,-2,-1,0,-1,-1,1,2,0,-1,-1,-2,-3,-2,1,-1,-4
L: -1,-4,-1,-4,-3,0,-4,-3,2,3,-2,4,2,-3,-1,-3,-2,-2,-2,-1,-1,1,-2,-1,-3,-1,-4
M: -1,-3,-1,-3,-2,0,-3,-2,1,2,-1,2,5,-2,-1,-2,0,-1,-1,-1,-1,1,-1,-1,-1,-1,-4
N: -2,3,-3,1,0,-3,0,1,-3,-3,0,-3,-2,6,-1,-2,0,0,1,0,-1,-3,-4,-2,0,-1,-4
O: 0,-1,-2,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-2,-1,-1,0,0,-1,-1,-2,-1,-1,-1,-4
P: -1,-2,-3,-1,-1,-4,-2,-2,-3,-3,-1,-3,-2,-2,-2,7,-1,-2,-1,-1,-2,-2,-4,-3,-1,-2,-4
Q: -1,0,-3,0,2,-3,-2,0,-3,-3,1,-2,0,0,-1,-1,5,1,0,-1,-1,-2,-2,-1,3,-1,-4
R: -1,-1,-3,-2,0,-3,-2,0,-3,-3,2,-2,-1,0,-1,-2,1,5,-1,-1,-1,-3,-3,-2,0,-1,-4
S: 1,0,-1,0,0,-2,0,-1,-2,-2,0,-2,-1,1,0,-1,0,-1,4,1,0,-2,-3,-2,0,0,-4
T: 0,-1,-1,-1,-1,-2,-2,-2,-1,-1,-1,-1,-1,0,0,-1,-1,-1,1,5,0,0,-2,-2,-1,0,-4
U: 0,-1,-2,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-2,-1,-1,0,0,-1,-1,-2,-1,-1,-1,-4
V: 0,-3,-1,-3,-2,-1,-3,-3,3,2,-2,1,1,-3,-1,-2,-2,-3,-2,0,-1,4,-3,-1,-2,-1,-4
W: -3,-4,-2,-4,-3,1,-2,-2,-3,-3,-3,-2,-1,-4,-2,-4,-2,-3,-3,-2,-2,-3,11,2,-3,-2,-4
Y: -2,-3,-2,-3,-2,3,-3,2,-1,-1,-2,-1,-1,-2,-1,-3,-1,-2,-2,-2,-1,-1,2,7,-2,-1,-4
Z: -1,1,-3,1,4,-3,-2,0,-3,-3,1,-3,-1,0,-1,-1,3,0,0,-1,-1,-2,-3,-2,4,-1,-4
X: 0,-1,-2,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-2,-1,-1,0,0,-1,-1,-2,-1,-1,-1,-4
*: -4,-4,-4,-4,-4,-4,-4,-4,-4,-4,-4,-4,-4,-4,-4,-4,-4,-4,-4,-4,-4,-4,-4,-4,-4,-4,1

Alignment Score: 317
Alignment Length: 81
#Match-Mismatch pairs: 402
Score contribution by match-mismatch pairs: 602
#Gap extensions: 51
Score contribution by gap extensions: -51
#Gap openings: 18
Score contribution by gap openings: -234

#Pairs: 
   A,B,C,D,E,F,G,H,I,J,K,L,M,N,O,P,Q,R,S,T,U,V,W,Y,Z,X,*,
A: 15,0,0,0,4,2,3,0,1,0,4,4,0,1,0,4,0,2,1,0,0,3,0,0,0,0,0
B: 0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
C: 0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
D: 2,0,0,3,4,0,0,0,1,0,3,2,0,2,0,0,2,2,1,1,0,0,0,0,0,0,0
E: 11,0,0,2,14,0,4,0,2,0,5,0,0,4,0,1,2,1,5,0,0,2,0,0,0,0,0
F: 1,0,0,0,0,3,0,0,1,0,0,1,0,1,0,0,0,2,1,0,0,1,1,3,0,0,0
G: 3,0,0,0,1,0,3,0,0,0,0,0,0,1,0,0,0,0,4,0,0,0,0,0,0,0,0
H: 1,0,0,0,0,1,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
I: 1,0,0,0,1,1,1,0,0,0,0,1,0,1,0,0,1,0,0,0,0,4,0,0,0,0,0
J: 0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
K: 4,0,0,2,2,0,0,0,1,0,30,0,0,2,0,0,2,3,3,1,0,2,0,0,0,0,0
L: 2,0,0,1,1,0,1,0,1,0,4,6,1,0,0,1,0,1,0,0,0,1,0,0,0,0,0
M: 0,0,0,0,1,2,0,0,2,0,3,4,2,2,0,1,0,1,0,0,0,2,0,0,0,0,0
N: 0,0,0,1,2,1,4,0,0,0,1,0,1,2,0,0,0,1,1,0,0,0,0,0,0,0,0
O: 0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
P: 2,0,0,0,2,0,0,0,0,0,0,0,0,0,0,9,0,0,0,0,0,0,0,0,0,0,0
Q: 5,0,0,0,2,0,2,0,0,0,0,0,0,0,0,0,0,0,1,1,0,0,0,0,0,0,0
R: 0,0,0,0,2,1,0,0,0,0,9,1,0,3,0,1,1,11,0,0,0,0,0,0,0,0,0
S: 8,0,0,0,5,1,3,0,0,0,3,2,1,6,0,1,1,0,6,0,0,1,0,0,0,0,0
T: 1,0,0,0,0,0,0,0,0,0,1,0,0,0,0,0,0,0,3,0,0,0,0,0,0,0,0
U: 0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
V: 3,0,0,0,1,0,1,0,0,0,0,1,1,0,0,1,1,0,0,1,0,2,0,0,0,0,0
W: 0,0,0,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,6,3,0,0,0
Y: 0,0,0,0,0,3,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,3,11,0,0,0
Z: 0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
X: 0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
*: 0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
