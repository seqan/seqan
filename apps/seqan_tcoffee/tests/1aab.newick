(hmgl_trybr,(hmgt_mouse,(hmgb_chite,hmgl_wheat)))
