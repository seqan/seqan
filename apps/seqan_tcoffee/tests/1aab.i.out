Scoring parameters:
*Gap opening: -13
*Gap extension: -1
*Scoring matrix: 
   A,R,N,D,C,Q,E,G,H,I,L,K,M,F,P,S,T,W,Y,V,B,Z,X,*,
A: 4,-1,-2,-2,0,-1,-1,0,-2,-1,-1,-1,-1,-2,-1,1,0,-3,-2,0,-2,-1,0,-4
R: -1,5,0,-2,-3,1,0,-2,0,-3,-2,2,-1,-3,-2,-1,-1,-3,-2,-3,-1,0,-1,-4
N: -2,0,6,1,-3,0,0,0,1,-3,-3,0,-2,-3,-2,1,0,-4,-2,-3,3,0,-1,-4
D: -2,-2,1,6,-3,0,2,-1,-1,-3,-4,-1,-3,-3,-1,0,-1,-4,-3,-3,4,1,-1,-4
C: 0,-3,-3,-3,9,-3,-4,-3,-3,-1,-1,-3,-1,-2,-3,-1,-1,-2,-2,-1,-3,-3,-2,-4
Q: -1,1,0,0,-3,5,2,-2,0,-3,-2,1,0,-3,-1,0,-1,-2,-1,-2,0,3,-1,-4
E: -1,0,0,2,-4,2,5,-2,0,-3,-3,1,-2,-3,-1,0,-1,-3,-2,-2,1,4,-1,-4
G: 0,-2,0,-1,-3,-2,-2,6,-2,-4,-4,-2,-3,-3,-2,0,-2,-2,-3,-3,-1,-2,-1,-4
H: -2,0,1,-1,-3,0,0,-2,8,-3,-3,-1,-2,-1,-2,-1,-2,-2,2,-3,0,0,-1,-4
I: -1,-3,-3,-3,-1,-3,-3,-4,-3,4,2,-3,1,0,-3,-2,-1,-3,-1,3,-3,-3,-1,-4
L: -1,-2,-3,-4,-1,-2,-3,-4,-3,2,4,-2,2,0,-3,-2,-1,-2,-1,1,-4,-3,-1,-4
K: -1,2,0,-1,-3,1,1,-2,-1,-3,-2,5,-1,-3,-1,0,-1,-3,-2,-2,0,1,-1,-4
M: -1,-1,-2,-3,-1,0,-2,-3,-2,1,2,-1,5,0,-2,-1,-1,-1,-1,1,-3,-1,-1,-4
F: -2,-3,-3,-3,-2,-3,-3,-3,-1,0,0,-3,0,6,-4,-2,-2,1,3,-1,-3,-3,-1,-4
P: -1,-2,-2,-1,-3,-1,-1,-2,-2,-3,-3,-1,-2,-4,7,-1,-1,-4,-3,-2,-2,-1,-2,-4
S: 1,-1,1,0,-1,0,0,0,-1,-2,-2,0,-1,-2,-1,4,1,-3,-2,-2,0,0,0,-4
T: 0,-1,0,-1,-1,-1,-1,-2,-2,-1,-1,-1,-1,-2,-1,1,5,-2,-2,0,-1,-1,0,-4
W: -3,-3,-4,-4,-2,-2,-3,-2,-2,-3,-2,-3,-1,1,-4,-3,-2,11,2,-3,-4,-3,-2,-4
Y: -2,-2,-2,-3,-2,-1,-2,-3,2,-1,-1,-2,-1,3,-3,-2,-2,2,7,-1,-3,-2,-1,-4
V: 0,-3,-3,-3,-1,-2,-2,-3,-3,3,1,-2,1,-1,-2,-2,0,-3,-1,4,-3,-2,-1,-4
B: -2,-1,3,4,-3,0,1,-1,0,-3,-4,0,-3,-3,-2,0,-1,-4,-3,-3,4,1,-1,-4
Z: -1,0,0,1,-3,3,4,-2,0,-3,-3,1,-1,-3,-1,0,-1,-3,-2,-2,1,4,-1,-4
X: 0,-1,-1,-1,-2,-1,-1,-1,-1,-1,-1,-1,-1,-1,-2,0,0,-2,-1,-1,-1,-1,-1,-4
*: -4,-4,-4,-4,-4,-4,-4,-4,-4,-4,-4,-4,-4,-4,-4,-4,-4,-4,-4,-4,-4,-4,-4,1

Alignment Score: 317
Alignment Length: 81
#Match-Mismatch pairs: 402
Score contribution by match-mismatch pairs: 602
#Gap extensions: 51
Score contribution by gap extensions: -51
#Gap openings: 18
Score contribution by gap openings: -234

#Pairs: 
   A,R,N,D,C,Q,E,G,H,I,L,K,M,F,P,S,T,W,Y,V,B,Z,X,*,
A: 15,2,1,0,0,0,4,3,0,1,4,4,0,2,4,1,0,0,0,3,0,0,0,0
R: 0,11,3,0,0,1,2,0,0,0,1,9,0,1,1,0,0,0,0,0,0,0,0,0
N: 0,1,2,1,0,0,2,4,0,0,0,1,1,1,0,1,0,0,0,0,0,0,0,0
D: 2,2,2,3,0,2,4,0,0,1,2,3,0,0,0,1,1,0,0,0,0,0,0,0
C: 0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
Q: 5,0,0,0,0,0,2,2,0,0,0,0,0,0,0,1,1,0,0,0,0,0,0,0
E: 11,1,4,2,0,2,14,4,0,2,0,5,0,0,1,5,0,0,0,2,0,0,0,0
G: 3,0,1,0,0,0,1,3,0,0,0,0,0,0,0,4,0,0,0,0,0,0,0,0
H: 1,0,0,0,0,0,0,0,0,1,0,0,0,1,0,0,0,0,0,0,0,0,0,0
I: 1,0,1,0,0,1,1,1,0,0,1,0,0,1,0,0,0,0,0,4,0,0,0,0
L: 2,1,0,1,0,0,1,1,0,1,6,4,1,0,1,0,0,0,0,1,0,0,0,0
K: 4,3,2,2,0,2,2,0,0,1,0,30,0,0,0,3,1,0,0,2,0,0,0,0
M: 0,1,2,0,0,0,1,0,0,2,4,3,2,2,1,0,0,0,0,2,0,0,0,0
F: 1,2,1,0,0,0,0,0,0,1,1,0,0,3,0,1,0,1,3,1,0,0,0,0
P: 2,0,0,0,0,0,2,0,0,0,0,0,0,0,9,0,0,0,0,0,0,0,0,0
S: 8,0,6,0,0,1,5,3,0,0,2,3,1,1,1,6,0,0,0,1,0,0,0,0
T: 1,0,0,0,0,0,0,0,0,0,0,1,0,0,0,3,0,0,0,0,0,0,0,0
W: 0,0,0,0,0,0,0,0,0,0,0,0,0,1,0,0,0,6,3,0,0,0,0,0
Y: 0,0,0,0,0,0,0,0,0,0,0,0,0,3,0,0,0,3,11,0,0,0,0,0
V: 3,0,0,0,0,1,1,1,0,0,1,0,1,0,1,0,1,0,0,2,0,0,0,0
B: 0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
Z: 0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
X: 0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
*: 0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
